client/src/components/relatedItems/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+
+
+export default class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+    // this.el = document.createElement('div');
+    this.state = {
+      modal: document.createElement('div')
+    }
+  }
+  componentDidMount() {
+    const modalRoot = document.getElementsByClassName('modal-container')[0];
+    modalRoot.appendChild(this.state.modal);
```

Comment:
I think you can call `document.querySelector('.modal-container');` and that way you don't need to get an array and pull the first element from the array like with `getElementsByClassName`

Reply:
I changed to it and it works well, I'll change in the next pull request.Thanks, Nate.